src/api/registrationApi.ts

```diff
@@ -64,6 +64,13 @@ export const addTicketMessage = async (ticketId: string, message: string) =>
     data: { message },
   });
 
+export const deleteTicketMessage = async (ticketId: string, messageId: string) => {
+  await authenticatedApiRequest2({
+    url: `${ticketId}/message/${messageId}`,
+    method: 'DELETE',
+  });
+};
```

Comment:
Tenker det er naturlig å returnere denne, selv om returverdi ikke skal bruke til noe nå. Litt for å aligne me alle andre kallene her også
```suggestion
export const deleteTicketMessage = async (ticketId: string, messageId: string) =>
  await authenticatedApiRequest2({
    url: `${ticketId}/message/${messageId}`,
    method: 'DELETE',
  });
```